front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -46,12 +47,13 @@ interface InputProps {
 }
 
 interface DataProps {
-  tenant: GetAppConfigurationChildProps;
-  passwordLoginEnabled: boolean | null;
-  googleLoginEnabled: boolean | null;
-  facebookLoginEnabled: boolean | null;
   azureAdLoginEnabled: boolean | null;
+  facebookLoginEnabled: boolean | null;
   franceconnectLoginEnabled: boolean | null;
+  googleLoginEnabled: boolean | null;
+  passwordLoginEnabled: boolean | null;
+  tenant: GetAppConfigurationChildProps;
+  viennaLoginEnabled: boolean | null;
```

Comment:
If you still have time and are up for the challenge, you can replace the resource components with hooks.

Reply:
We are a bit on a timer with Vienna here, so I'd prefer not to at this time.